powa/static/js/plugins/vuetify.js

```diff
@@ -113,12 +119,26 @@ function createVuetify() {
       iconfont: "mdiSvg",
     },
     theme: {
+      options: {
+        customProperties: true,
+      },
```

Comment:
Can you please elaborate why this is needed?

Reply:
It's needed to use css variable in vuetify [custom properties](https://v2.vuetifyjs.com/en/features/theme/#custom-properties)